organizing_hub/views/call_views.py

```diff
@@ -292,6 +340,17 @@ def form_valid(self, form):
         """Set local group and owner before save"""
         form.instance.local_group = self.get_local_group()
         form.instance.owner = self.request.user.callprofile
+
+        """Take comma separated caller emails and get or create corresponding
+        call profiles"""
+        caller_emails = form.cleaned_data['caller_emails']
+
+        """Save call_campaign because we need a primary key in order to save
+        Many-to-many caller relationships"""
+        call_campaign = form.save()
+        caller_ids = get_or_create_callers(caller_emails)
```

Comment:
Maybe better to put this in a post save signal?

Reply:
I don't think it makes sense to put in post_save sense it only relates to this form really. Wouldn't want to do this logic after an admin form save, for example.